flink-ml-python/pyflink/ml/lib/tests/test_ml_lib_completeness.py

```diff
@@ -140,6 +145,11 @@ def module(self):
         from pyflink.ml.lib import stats
         return stats
 
+    def exclude_java_stage(self) -> List[str]:
+        return [
+            "org.apache.flink.ml.stats.anovatest.ANOVATest",
```

Comment:
We can simplify it to `"anovatest.ANOVATest"` but not `"ANOVATest"`, because it is not required to keep the package name equalsIgnoreCase the class name, that's why we have both `_java_transformer_package_name` and `_java_transformer_class_name` in  class `JavaFeatureTransformer`.